ocaml/xapi/repository.ml

```diff
@@ -695,18 +654,7 @@ let apply_livepatches' ~__context ~host ~livepatches =
     )
     livepatches
 
-let apply_updates' ~__context ~host ~updates_info ~livepatches ~acc_rpm_updates
-    =
-  (* This function runs on coordinator host *)
-  (* Install RPM updates firstly *)
-  Helpers.call_api_functions ~__context (fun rpc session_id ->
-      Client.Client.Repository.apply ~rpc ~session_id ~host
-  ) ;
-  (* Apply live patches with best effort *)
-  let successful_livepatches, failed_livepatches =
-    apply_livepatches' ~__context ~host ~livepatches
-  in
```

Comment:
Sorry, may I know if there is any change on how livepatches are applied, seems the function `apply_livepatches` is not changed, but the comment changed from
(* Apply live patches with best effort *)
to
(* Apply livepatches *)